src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence.mdx

```diff
@@ -12,36 +12,103 @@ redirects:
 
 If you haven't already, [sign up for a New Relic account](https://newrelic.com/signup) to get started.
 
 <Callout variant="important">
   Pathways have been deprecated, and are replaced by [workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
 
-  If you still have access to Pathways on New Relic One, reach out to your New Relic account representative and ask to be upgraded.
+  If you still have access to pathways, reach out to your New Relic account representative and ask to be upgraded.
 </Callout>
 
 ## Set up incident intelligence [#get-started]
 
-To enable incident intelligence, follow these steps. Afterwards, issues should start to appear in your issue feed.
+To enable incident intelligence and start seeing issues in  your issue feed, follow these steps.
 
 * [1. Configure your environment (one-time)](#1-configure-environment).
 * [2. Configure sources](#1-configure-sources).
 * [3. Configure destinations](#2-configure-destinations).
 * [4. Configure workflows](#3-configure-workflows).
 
-## 1. Configure your environment (one-time) [#1-configure-environment]
+## 1. Configure your environment [#1-configure-environment]
+
+An environment groups together a set of one or more accounts. This allows you to see cross-account actions such as correlation, configuration, and data in one place. Everyone on your team needs access to your environment account and be authorized to view the data in it.
+
+By default, when you create an account, it's added to its own environment. 
+
+If you need cross-account collaboration, you can add multiple accounts into a single custom environment. You should only do this if you need to track cross-account data.
+
+By default, each account is in its own environment if it was created after November 2021. This is the recommended setting. 
+
+### What's stored in the environment? [#environment-detail]
+
+<img
+  title="The incident intelligence environment."
+  alt="A screenshot of the incident intelligence environment dropdown."
+  src={environment}
+/>
+
+<figcaption>
+  Use the environment dropdown to see your cross-account applied intelligence activity.
+</figcaption>
+
+* Your workflows configuration
+* Your decisions configuration
+* Your incidents
+* Issues created from your incidents
 
-To set up an environment in incident intelligence, you need an administrator to select a New Relic account for it. This account should be the one your team is using.
+Workflow enricher queries are executed at the environment account level.
 
-* **Who sets the environment?** Only administrators, and only for accounts where they have admin privileges.
-* **Can administrators set more than one environment?** They can set one environment per parent account and its child accounts. More than one can be set if an administrator has privileges for more than one parent account.
```

Comment:
'parent accounts' are not really a concept for v2 user model, and are much more a v1 user model thing. basically on v2 user model, the first account created in an organization is technically the parent account and other accounts in there are its children accounts, but we don't really talk about that as it no longer has much application to customers. I will add this context about parent accounts to this glossary entry https://docs.newrelic.com/docs/new-relic-solutions/get-started/glossary/#parent-account and maybe you can just link to it so if people are confused they can read up on it more.

Reply:
Sorry, you spent time commenting on deleted content...